reason-react-native/src/apis/PanResponder.md

```diff
@@ -1,9 +1,137 @@
 ---
-id: apis/Packager
-title: Packager
-wip: true
+id: apis/PanResponder
+title: PanResponder
 ---
 
+`PanResponder` allows more fine-grained control of the _Gesture Responder
+System_, as the handlers can depend also on `gestureState`, in addition to the
+`Event.pressEvent` which are passed to the _Gesture Responder System_ handlers.
+In essence, by means of a custom `PanResponder`, _Gesture Responder System_
+handlers are extended to also depend upon the `gestureState`.
+
+## Methods
+
+- `create` allows creation of a `PanResponder` given a `config`
+
+```reason
+create(config)
+```
+
+- `panHandlers` is a getter method to access `panHandlers` within a
+  `PanResponder`
+
+```reason
+panHandlers(t)
+```
+
+- `onMoveShouldSetResponder` is a getter method to access the
+  `onMoveShouldSetResponder` callback of a `panHandlers` object
+
+```reason
+onMoveShouldSetResponder(panHandlers)
+```
+
+- `onMoveShouldSetResponderCapture` is a getter method to access the
+  `onMoveShouldSetResponderCapture` callback of a `panHandlers` object
+
+```reason
+onMoveShouldSetResponderCapture(panHandlers)
+```
+
+- `onStartShouldSetResponder` is a getter method to access the
+  `onStartShouldSetResponder` callback of a `panHandlers` object
+
+```reason
+onStartShouldSetResponder(panHandlers)
+```
+
+- `onStartShouldSetResponderCapture` is a getter method to access the
+  `onStartShouldSetResponderCapture` callback of a `panHandlers` object
+
+```reason
+onStartShouldSetResponderCapture(panHandlers)
+```
+
+- `onResponderReject` is a getter method to access the `onResponderReject`
+  callback of a `panHandlers` object
+
+```reason
+onResponderReject(panHandlers)
+```
+
+- `onResponderGrant` is a getter method to access the `onResponderGrant`
+  callback of a `panHandlers` object
+
+```reason
+onResponderGrant(panHandlers)
+```
+
+- `onResponderRelease` is a getter method to access the `onResponderRelease`
+  callback of a `panHandlers` object
+
+```reason
+onResponderRelease(panHandlers)
+```
+
+- `onResponderMove` is a getter method to access the `onResponderMove` callback
+  of a `panHandlers` object
+
+```reason
+onResponderMove(panHandlers)
+```
+
+- `onResponderTerminate` is a getter method to access the `onResponderTerminate`
+  callback of a `panHandlers` object
+
+```reason
+onResponderTerminate(panHandlers)
+```
+
+- `onResponderStart` is a getter method to access the `onResponderStart`
+  callback of a `panHandlers` object
+
+```reason
+onResponderStart(panHandlers)
+```
+
+- `onResponderTerminationRequest` is a getter method to access the
+  `onResponderTerminationRequest` callback of a `panHandlers` object
+
+```reason
+onResponderTerminationRequest(panHandlers)
+```
+
+- `onResponderEnd` is a getter method to access the `onResponderEnd` callback of
+  a `panHandlers` object
+
+```reason
+onResponderEnd(panHandlers)
+```
+
+## Types
+
+- `Event.pressEvent` is `Event.syntheticEvent(pressEventPayload)` where the
```

Comment:
I think we shouldn't doc other module like this & instead make links to ease maintenance.
You could move that to `Event.md` & make a link to it via the expected anchor link?

Reply:
Perhaps replace this with a link to `Event.md` when that's ready?

Reply:
Yeah you can link to `/bs-react-native/en/docs/apis/Event/` (not ideal but I will deal with this problem later)